server/testutils/data.go

```diff
@@ -136,7 +136,7 @@ func GetChannelLinks(count int) []*storemodels.ChannelLink {
 func GetFileInfo() *model.FileInfo {
 	return &model.FileInfo{
 		Id:       GetID(),
-		Name:     "mockFileName",
+		Name:     "mockFile.Name.txt",
```

Comment:
```suggestion
		Name:     "mockFileName.txt",
```

Reply:
this was done to test whether we are properly fetching the file extension